### ISSUE TITLE
poc: Use binary encoding for SHA-3 inputs rather than ascii

### ISSUE THREAD

**Issue by cjpatton** [jimouris/draft-mouris-cfrg-mastic#17]

**Comment by divergentdave** [comment]
Agreed, an encoding for prefixes will be needed eventually for the aggregation parameter, and we may as well use it here. Plus, using ASCII may require implementations to use a bigint library for sufficiently long inputs.

**Comment by cjpatton** [comment]
@divergentdave please give a quick look at #46.
